### ISSUE TITLE
Stuck on Bridge

### ISSUE THREAD

**Issue by Woodwag** [bob-collective/bob#369]
Moving funds from BOB to ETH on the BOB Bridge
Withdraw from BOB is done
State root published
Is stuck on Prove, push the prove button and it instantly come up with "Failed  to submit proof"
Help?

**Comment by derrekcoleman** [comment]
Hey @Woodwag 👋 Sorry to hear the withdrawal is giving you trouble. I've had the same problem on a different chain, so I know how frustrating it can be. 

Have you opened a support ticket in [our Discord](https://discord.com/invite/gobob)?

**Comment by Woodwag** [comment]
Hey derrek, thanks for your reply. Tried to get on Discord, but even with invite it won't allow me to join, and I'm not that good with computers to work out what's wrong. I was hoping that the transaction would not PROVE because of the 7 day challenge period, but this is now passed and no luck. Once again any help would be great

**Comment by derrekcoleman** [comment]
Shoot, getting you into Discord is important so that we can privately ask follow-up questions without exposing your information publicly. Please be very careful when asking for support with crypto-related things - scammers will impersonate support team members and try to trick you into clicking on malicious links, divulging information like your seed phrase, etc.

Please try opening that discord link (after verifying for yourself that it is actually a discord link and I'm not tricking you 😉 ) in another browser, an incognito window, or after installing the desktop version of Discord.
@Woodwag

**Comment by Woodwag** [comment]
derrek, is Step 1. Go to : https://help.helpsolve.io/  a safe option? I opened a ticket a got 3 or 4 people trying to help, very confusing, everyone has different opinion

**Comment by derrekcoleman** [comment]
@Woodwag no, that is not safe. That's a good example of what I'm referring to. Anyone reaching out to you directly is, unfortunately, very likely trying to scam you.

Have you had any luck opening the Discord invite I sent above?

**Comment by Woodwag** [comment]
Derek, l want to thank you for your help, can still not get onto discord with your invite link.
So based on the fact that the money stuck is only $500 and there is too much risk with scam etc, I'm going take the loss.
It's a pity as l think BOB can be big that's why l put some ETH into the first stage. Maybe one day someone will fix the issue and the transaction will go through. Once again thank you for your help

**Comment by derrekcoleman** [comment]
@Woodwag not so fast, we aren't going to leave you high and dry. Could you send an email to derrek [at] gobob.xyz and we'll take it from there?

**Comment by Woodwag** [comment]
How can you close something that has not been fixed?

**Comment by derrekcoleman** [comment]
@Woodwag, please email me at derrek [at] gobob.xyz so we can continue the diagnosis in a less public forum
